app/helpers/pagy_helper.rb

```diff
@@ -4,28 +4,40 @@ module PagyHelper
   def pagy_nav_govuk(pagy)
     return "" if pagy.pages == 1
 
-    link   = pagy_link_proc(pagy, link_extra: "class='govuk-link'")
+    link   = pagy_link_proc(pagy, link_extra: "class='govuk-link govuk-pagination__link'")
```

Comment:
This method now contains a lot of HTML. Wondering if it is possible to use a partial?

Reply:
I have looked in to this and using a partial in the helper method is not conventional, taking the html out and using it in the views looks like a mess and doesn't really work. However, if we want to take the helper out completely and create a shared partial within the application views it'll be changing a lot of the application code as it is used in many places. I can do it but I'll create a separate branch for that. Can we approve this for now?